pages/registration.js

```diff
@@ -19,7 +20,8 @@ const Registration = () => {
 
     try {
       setLoading(true);
-      await signup(e.target.email.value, e.target.passwd.value);
+      const newUser = await signup(e.target.email.value, e.target.passwd.value);
+      addNewUser({ id: newUser.user.uid, email: e.target.email.value });
       router.push('/');
     } catch (err) {
```

Comment:
It is time for you to remember to separate the code nicely, with new lines. Please keep that in mind.